### PR TITLE
Rename groups in params.yaml.

### PR DESCRIPTION
**Pull request by MaxBlesch**
### What problem do you want to solve?

The group names of the variables are not self-explanatory at the moment. We rename the groups according to https://github.com/iza-institute-of-labor-economics/gettsim/issues/79#issuecomment-554283492

### Todo

- [x] Rename groups in params.yaml.

### CODE REVIEW

**Comment by MaxBlesch** [comment]
> I was mistaken. The `ahunt` items refer to "unterhalt" for children, which we don't have yet as group. We are not using them now, but it's no harm to keep them in.

I would therefore name kindesunterhalt?

File: gettsim/data/param.yaml [hunk 1]
```diff
@@ -1073,7 +1073,7 @@ ahsatz0:
       value: 0
       note:
 ahsatz1:
-  group: alg
+  group: arbeitsl_geld
```
[the remark this answers is not in the document]
Comment (MaxBlesch) [reply gettsim/data/param.yaml]: Changed to arbeitsl_hilfe .

File: gettsim/data/param.yaml [hunk 2]
```diff
@@ -1055,7 +1055,7 @@ agsatz1:
       value: .67
       note: Prüfen!
 ahsatz0:
-  group: alg
+  group: arbeitsl_geld
   name:
```
[the remark this answers is not in the document]
Comment (MaxBlesch) [reply gettsim/data/param.yaml]: Changed to arbeitsl_hilfe .

**Comment by hmgaudecker** [comment]
> > I was mistaken. The `ahunt` items refer to "unterhalt" for children, which we don't have yet as group. We are not using them now, but it's no harm to keep them in.
> 
> I would therefore name kindesunterhalt?

Can't we merge the `unterh_vorsch` and `kindesunterhalt` categories?  I am a bit reluctant to add too many groups.

Sure its different payors, but you are looking for the code in the same places and I would suppose they are also closely related. @Eric-Sommer, @mjbloemer ?

**Comment by Eric-Sommer** [comment]
We could merge them into one group. The major use case for the `ahunt` stuff I see is that it reduces taxable income for the payer, so not really the same place in the code.

**Comment by Eric-Sommer** [comment]
> I would thing that "unterhalt" is the overarching category?

I would agree

**Pull request merged**